info.hiergiltdiestfu.aws.neptune.graphml/src/main/java/org/graphdrawing/graphml/xmlns/EdgeType.java

```diff
@@ -108,7 +108,10 @@ public String getDesc() {
     public void setDesc(String value) {
         this.desc = value;
     }
-
+    
+    public String toString() {
+    	return "Objekt ist da" + this.id;
+    }

```

Comment:
Manuelle Ergänzungen sind in generierten Klassen nur wenig sinnstiftend. Die Faustregel ist, dass Generate jederzeit regenerierbar sein müssen. Wenn Du Ergänzungen haben willst, brauchst Du entweder einen Wrapper, oder eine Änderung des Generators.